cmd/root/root.go

```diff
@@ -52,13 +52,16 @@ func CreateRootCommand() *cobra.Command {
 	tmp.Flags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.pdk.yaml)")
 
 	tmp.PersistentFlags().StringVar(&LogLevel, "log-level", zerolog.InfoLevel.String(), "Log level (debug, info, warn, error, fatal, panic)")
-	tmp.RegisterFlagCompletionFunc("log-level", func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) { //nolint:errcheck
+	err := tmp.RegisterFlagCompletionFunc("log-level", func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
 		if len(args) != 0 {
 			return nil, cobra.ShellCompDirectiveNoFileComp
 		}
 		var levels = []string{"debug", "info", "warn", "error", "fatal", "panic"}
 		return utils.Find(levels, toComplete), cobra.ShellCompDirectiveNoSpace | cobra.ShellCompDirectiveNoFileComp
 	})
+	if err != nil {
```

Comment:
Instead of panicing, can we use cobra.CheckErr somehow? These are build up before the line https://github.com/puppetlabs/pdkgo/blob/main/main.go#L82 and won't get the same error handling as the rest of the program.

Reply:
yep i can switch that